app/src/main/java/de/westnordost/streetcomplete/quests/camping/CampType.kt

```diff
@@ -0,0 +1,8 @@
+package de.westnordost.streetcomplete.quests.camping
+
+enum class CampType(val tents: Boolean, val caravans: Boolean) {
+    TENTS_AND_CARAVANS(true, true),
+    TENTS_ONLY(true, false),
+    CARAVANS_ONLY(false, true),
+    BACKCOUNTRY(false, false),
```

Comment:
At the minimum I'd suggest this should have a comment added
```suggestion
    // Only the enum value is used here, the tent and caravan values are ignored.
    BACKCOUNTRY(false, false),
```